contracts/protocol/tokenization/StaticATokenLM.sol

```diff
@@ -473,6 +479,18 @@ contract StaticATokenLM is
     _updateUserSnapshotRewardsPerToken(user, currentRewardsIndex);
   }
 
+  function _updateL2TokenState() internal {
+    (bool success, ) = _l1TokenBridge.call(
```

Comment:
Which is the point to do so low-level call instead of defining an interface?

Reply:
No real reason, I'll use an interface instead